manifests/backports.pp

```diff
@@ -0,0 +1,41 @@
+class apt::backports (
+  $location = undef,
+  $release  = undef,
+  $repos    = undef,
+  $key      = undef,
+  $pin      = 200,
+){
+
+  if ($::apt::xfacts['lsbdistid'] == 'debian' or
+      $::apt::xfacts['lsbdistid'] == 'ubuntu') {
+    $_location = $::apt::backports['location']
+    $_release  = "${::apt::xfacts['lsbdistcodename']}-backports"
+    $_repos    = $::apt::backports['repos']
+    $_key      = $::apt::backports['key']
```

Comment:
I think we should probably have these be the default values, but still allow the user to pass custom location, release, repos, or key? Right now this just ignores the params if you're on debian or ubuntu.